collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -322,4 +333,40 @@ public PermissionsChecker providesPermissionChecker(@NonNull Context context) {
 
         return geoDependencyComponent;
     }
+
+    @NonNull
+    @Override
+    public OsmDroidDependencyComponent getOsmDroidDependencyComponent() {
```

Comment:
We have more and more builders like this one I think it's time to start keeping them in separate classes to avoid blowing this class?